tests/apps/courses/test_templates_course_detail.py

```diff
@@ -443,6 +447,95 @@ def test_templates_course_detail_no_programs(self):
         self.assertEqual(response.status_code, 200)
         self.assertNotContains(response, "course-detail__programs")
 
+    def test_templates_course_detail_teaser_empty_cover_empty(self):
+        """
+        Without video in teaser and no image in cover placeholders,
+        no component should be present on the teaser placeholder.
```

Comment:
Maybe use the real placeholder names for clarity? Same for other tests.
```suggestion
        Without video in `course_teaser` placeholder and no image in `course_cover` placeholder,
        no component should be present on the `course_teaser` placeholder.
```